modules/distr-tracing-tempo-troubleshoot-collecting-diagnostic-data-from-command-line.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="collecting-diagnostic-data-from-command-line_{context}"]
+= Collecting diagnostic data from the command line
+
+When submitting a support case, it is helpful to include diagnostic information about your cluster to Red Hat Support.
+You can use the `oc adm must-gather` tool to gather diagnostic data for resources of various types, such as `TempoStack` or `TempoMonolithic`, and `Deployment`, `Pod`, `ConfigMap`, and so on.
+The `oc adm must-gather` tool creates a new pod that collects these data.
+
+.Procedure
+
+* From the directory where you want to save the collected data, run the `oc adm must-gather` command to collect the data:
++
+[source,terminal]
+----
+$ oc adm must-gather --image=ghcr.io/grafana/tempo-operator/must-gather -- /usr/bin/must-gather --operator-namespace openshift-tempo-operator
```

Comment:
How about mentioning something about this command is using the default namespace where the operator is installed but, if you installed it in a different namespace you need to replace `openshift-tempo-operator` by the namespace where the operator was instaled?

Reply:
@iblancasa, should I add a similar fix for the OTEL page?